src/main/java/beans/SubmitResponseBean.java

```diff
@@ -53,6 +38,6 @@ public void setStatus(String status) {
 
     @Override
     public String toString(){
-        return "CurrentResponseBean: [sessionId=" + sessionId + "]";
+        return "SubmitResponseBean: [sessionId=" + sessionId + ",output= " + output + "]";
```

Comment:
seems like these `toString` functions could be some sort of utility where you pass the class and args and then generate a toString()